<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PoundageWidget</class>
 <widget class="QWidget" name="PoundageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>510</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>770</width>
     <height>360</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="title_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>28</y>
     <width>200</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>手续费承兑单</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_coinType">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_publishPoundage">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>109</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI Light</family>
     <pointsize>-1</pointsize>
     <weight>6</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 50 12px &quot;Microsoft YaHei UI Light&quot;;</string>
   </property>
   <property name="text">
    <string>发布承兑单</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_sortType">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>发布时间由早到晚</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>发布时间由晚到早</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_allPoundage">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>70</width>
     <height>18</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>所有</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_myPoundage">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>70</width>
     <height>18</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>我发布的</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
